com.unity.multiplayer.mlapi/Editor/NetworkAnimatorEditor.cs

```diff
@@ -1,94 +0,0 @@
-using System;
```

Comment:
Why are we removing this?

Reply:
Oh, because I removed the ability to select the parameters to sync (all of them sync now, so there's no need for this custom editor)